### PR TITLE
Cater for change in data.table v1.11.0

### PR DESCRIPTION
**Pull request by mattdowle**
Dear SpaDES.core team,

Thanks for using data.table!  Sorry for the inconvenience on this one.

Reverse dependency testing revealed `SpaDES.core` will break with the data.table v1.11.0 update to CRAN.
Bug fix 8 in v1.11.0 NEWS is :
> 8. Printing a null `data.table` with `print` no longer visibly outputs `NULL`, [#1852](https://github.com/Rdatatable/data.table/issues/1852). Thanks @aaronmcdaid for spotting and @MichaelChirico for the PR.

This PR should make SpaDES.core pass R CMD check with data.table both before and after v1.11.0

The failure is :
```
── 1. Failure: simList object initializes correctly (@test-simList.R#17)  ──────────────────────────
length(out) not equal to 75.
1/1 mismatches
[1] 73 - 75 == -2
══ testthat results  ═══════════════════════════════════════════════════════════
OK: 349 SKIPPED: 33 FAILED: 1
1. Failure: simList object initializes correctly (@test-simList.R#17) 
```

### CODE REVIEW

File: tests/testthat/test-simList.R [hunk 1]
```diff
@@ -14,7 +14,8 @@ test_that("simList object initializes correctly", {
   w <- getOption("width")
   options(width = 100L)
   out <- utils::capture.output(show(mySim))
-  expect_equal(length(out), 75)
+  nline <- if (out[60]=="NULL") 75 else 73 # data.table v1.11.0 no longer prints "NULL" data.table (bug fix 8 in NEWS)
```
Comment (MichaelChirico) [inline tests/testthat/test-simList.R]: maybe tag which version of `NEWS` for future reference

**Comment by achubaty** [comment]
Thank you for the head's up and for the PR. Can you please change the target branch to "development"?

File: tests/testthat/test-simList.R (hunk 1 above)
Comment (mattdowle) [reply tests/testthat/test-simList.R]: The comment on that line includes v1.11.0 ?

**Comment by achubaty** [comment]
thank you!

**Pull request merged**